### PR TITLE
[System] [Network] Add dimensions and clean up duplicated fields definition

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## What does this PR do?

This is a follow up PR of https://github.com/elastic/integrations/pull/6118#issuecomment-1569749682
In this PR:
- added dimensions to the network data_stream: additionally to the [default list](https://github.com/elastic/integrations/issues/5193#issuecomment-1536090359) of dimensions, must be added `system.network.name`
- cleaned up duplicated fields

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Relates https://github.com/elastic/integrations/pull/6118

### CODE REVIEW

File: packages/system/data_stream/network/fields/agent.yml [hunk 1]
```diff
@@ -195,26 +195,3 @@
       example: "stretch"
       description: >
         OS codename, if any.
-
-    - name: network.in.bytes
```
Comment (tetianakravchenko) [inline packages/system/data_stream/network/fields/agent.yml:199]: those fields are repeated in `fields/` twice: in `agent.yml` and in `fields.yml`

File: packages/system/data_stream/network/fields/base-fields.yml [hunk 2]
```diff
@@ -7,9 +7,6 @@
 - name: data_stream.namespace
   type: constant_keyword
   description: Data stream namespace.
-- name: '@timestamp'
```
Comment (tetianakravchenko) [inline packages/system/data_stream/network/fields/base-fields.yml:10]: repeated in `ecs.yml` https://github.com/elastic/integrations/pull/6405/files#diff-b76aed736a60d76fd3ebe52357896a07a3f735ea06b403aafc104a9b8f3b8f7dL2

File: packages/system/data_stream/network/fields/fields.yml [hunk 3]
```diff
@@ -53,25 +54,25 @@
   type: group
   fields:
     - name: network.in.bytes
-      type: scaled_float
```
Comment (tetianakravchenko) [inline packages/system/data_stream/network/fields/fields.yml:56]: see this comment for reference - https://github.com/elastic/integrations/pull/6118#discussion_r1211342091

File: packages/system/changelog.yml [hunk 4]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.30.0"
```
Comment (ruflin) [inline packages/system/changelog.yml:2]: Same as other PR, conflicting version number?

File: packages/system/data_stream/network/fields/fields.yml (hunk 3 above)
Comment (ruflin) [reply packages/system/data_stream/network/fields/fields.yml:56]: My understanding based on the previous comments is that the only affect this has is on the docs page, everything else already had `long`. Correct?

**Review by ruflin — COMMENTED** [review]
Change LGTM. 

But see my comment around package version.

File: packages/system/data_stream/network/fields/fields.yml (hunk 3 above)
Comment (tetianakravchenko) [reply packages/system/data_stream/network/fields/fields.yml:56]: Yes, I have the same understanding, all the `host.network.in.*`, `host.network.out.*` are of type `long` when instaling package:
Screenshot 2023-06-01 at 10 12 33

File: packages/system/changelog.yml (hunk 4 above)
Comment (tetianakravchenko) [reply packages/system/changelog.yml]: fixed in https://github.com/elastic/integrations/pull/6405/commits/57547c719bb02ae882a3daa290d41f29a9dfcbcb

**Review by ruflin — APPROVED** [review]

**Review by leehinman — APPROVED** [review]

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**